### ISSUE TITLE
Improving display of results

### ISSUE THREAD

**Issue by bkjproductions** [mdsimpson/contact-form-7-to-database-extension#99]
Not knocking the display of the results, but this simple bit of CSS might make the table easier to read:

```
table#cf2dbtable td {
    overflow: hidden;
    text-overflow: ellipsis;
    white-space: nowrap;
    vertical-align: top;
}
```

**Comment by kherasymenko** [comment]
> Not knocking the display of the results, but this simple bit of CSS might make the table easier to read:
> 
> ```
> table#cf2dbtable td {
>     overflow: hidden;
>     text-overflow: ellipsis;
>     white-space: nowrap;
>     vertical-align: top;
> }
> ```

Do I need add this style to end of file contact-form-7-to-database-extension/css/misctable.css ?

Thanks in advance!

**Comment by bkjproductions** [comment]
Yes, that would do it, but if the plugin is ever updated, you would need to update this again, probably.
Also try this, if you have fields that are textarea---that is, it seems the data is wrapped within each TD with a DIV:

```
table#cf2dbtable td div {
	height: 1.5em;
	 overflow: hidden;
    text-overflow: ellipsis;
    white-space: nowrap;
    vertical-align: top;
	max-width: 15em;
}
```
